src/tests/ftest/pool/permission.yaml

```diff
@@ -19,8 +19,6 @@ pool:
     name: daos_server
 createtests:
     createmode: !mux
-        mode_RO:
-            mode: 73
```

Comment:
Is 73 a valid mode?   If not should the test code conditional be removed all together?
``        if createmode == 73:
            permissions = 0
            expected_result = RESULT_FAIL``

Reply:
please see the update.

Reply:
I see it is expected to fail...